containerd-shim-v2/service.go

```diff
@@ -298,12 +299,19 @@ func (s *service) Start(ctx context.Context, r *taskAPI.StartRequest) (*taskAPI.
 		}
 
 		return &taskAPI.StartResponse{
-			Pid: PID,
+			Pid: s.pid,
 		}, nil
 	}
 
 	//start an exec
-	return nil, errdefs.ErrNotImplemented
+	_, err = startExec(ctx, s, r.ID, r.ExecID)
```

Comment:
Please use if/else here, this way you can factorize by calling:
```go
        return &taskAPI.StartResponse{
		Pid: s.pid,
	}, nil
```
only once at the end.

Reply:
Done